webpack/InsightsVulnerabilityHostIndexExtensions/CVECountCell.js

```diff
@@ -3,25 +3,31 @@ import PropTypes from 'prop-types';
 import { UnknownIcon } from '@patternfly/react-icons';
 import { Link } from 'react-router-dom';
 import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
-
 import { insightsCloudUrl } from '../InsightsCloudSync/InsightsCloudSyncHelpers';
+import { useAdvisorEngineConfig } from '../common/Hooks/ConfigHooks';
 
 const vulnerabilityApiPath = path =>
   insightsCloudUrl(`api/vulnerability/v1/${path}`);
 
 export const CVECountCell = ({ hostDetails }) => {
+  const isIopEnabled = useAdvisorEngineConfig();
+
   // eslint-disable-next-line camelcase
   const uuid = hostDetails?.subscription_facet_attributes?.uuid;
 
   const key = `HOST_CVE_COUNT_${uuid}`;
   const response = useAPI(
-    uuid ? 'get' : null,
+    isIopEnabled && uuid ? 'get' : null,
     vulnerabilityApiPath(`systems?uuid=${uuid}`),
     {
       key,
     }
   );
 
+  if (!isIopEnabled) {
```

Comment:
How about move this up to the line right after where `isIopEnabled` is defined?

Reply:
Fixed